### DISCUSSION TITLE
Question about the test

### DISCUSSION THREAD

**Discussion by ruizhang0101** [PaddlePaddle/PaddleSpatial#108] [BaiduKDDCup2022] [answered]
Dear host,
I am confused about the evaluation method. Now, I suppose that given a 14-days time series with stride 10 minutes, we should predict the next two days Patv with stride 10 minutes.
But the score function in the report confuses me. Why is there a sum for r=0 to r=k of the /delta t  with test instance k? The report said /delta  t_k is randomly sampled from [1,10], then what is the point about the /sum?

**Comment by zhoujingbo** [comment, marked as answer]
Hi，
At first, given a 14-days time series， the **time step** is 10 minutes, not the stride time.

To evaluate the model performance, using one-time point data set is not enough (for example, if we evaluate the model using the data starting at 8:00AM in May 27th is not enough).  

We need to slide the time series to generate the test data (that means, we should evaluate the model using data starting at like 8:00AM at May 27th, 8:50 AM at May 27th, 9:10 AM at May 27th ...). 

The interval between the starting time for test data is stride time (in the above example, the stride time interval is 50 minutes, 30 minutes, ...)

Then we sum the prediction error with different stride times to evaluate the performance of the model.

Hope this is helpful to you.

Thanks a lot

**Reply by ruizhang0101** [reply]
Thank you for your detailed reply.
I am wondering if it means that we should predict data more than 2 days, since the start time of evaluation is not fixed and random.
Or do you mean the given 14*144 time series starts at different time and we should only predict the next 288 Patv right after the given series.
